.github/workflows/eol.yml

```diff
@@ -0,0 +1,48 @@
+name: EOL check
+on: [push, pull_request]
```

Comment:
I think we need this to run on a schedule too, for the issue creation? And if EOL dates move.